public/locales/de/editor.json

```
@@ -0,0 +1,16 @@
+{
+  "editor-desktop-optimized": "Diese Seite ist für Desktop optimiert",
+  "editor-visit-desktop": "Bitte besuche readme.so auf einem Desktop um deine Readme zu erstellen!",
+  "nav-download": "Herunterladen",
+  "download-readme-generated": "Readme generiert!",
+  "download-reach-out": "Danke, dass du readme.so verwendest! Fühl dich frei, mich über",
+  "download-feedback": "für Rückmeldungen zu kontaktieren.",
+  "download-coffee": "Wenn du dieses Produkt hilfreich findest, unterstütze mich doch gerne mit einer Tasse Kaffe!",
+  "editor-column-editor": "Editor",
+  "editor-select": "Wähle einen Abschnitt aus der linken Seitennavigation um den Inhalt zu bearbeiten",
+  "preview-column-preview": "Vorschau",
+  "preview-column-raw": "Unbearbeitet",
+  "section-column-section": "Abschnitte",
+  "section-column-click-edit": "Klicke auf einen Abschnitt unterhalb, um den Inhalt zu bearbeiten",
+  "section-column-click-add": "Klicke auf einen Abschnitt unterhalb, um ihn zu deiner Readme hinzuzufügen"
```

```suggestion
  "section-column-click-edit": "Klicke auf einen der unteren Abschnitte, um den Inhalt zu bearbeiten",
  "section-column-click-add": "Klicke auf einen der unteren Abschnitte, um ihn zu deiner Readme hinzuzufügen"
```

Here I also think it sounds more natural.